src/components/Questions/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+import DateInput from "../DateInput/index.jsx";
+import TextInput from "../TextInput/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const Questions = () => {
```

Comment:
I feel this component is redundant. Maybe you could just render each question component in your routes as a children of the main layout instead. And each question should have a component. Something like `PersonalInfo.tsx`, `ContactInfo.tsx`, etc.

Reply:
got It Thaaaanks ^^